source/standards/storing-credentials.html.md.erb

```diff
@@ -0,0 +1,48 @@
+---
+title: How to store credentials
+last_reviewed_on: 2021-08-06
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+Depending on how you [manage your accounts](/standards/accounts-with-third-parties.html), you, your team and the service you run may have credentials or other secrets that you need to store securely.
+
+## Personal credentials
+
+Examples include login details for:
+
+- GitHub
+- AWS
+- GOV.UK Signon
+
+[Use the password manager built into to your browser in the first instance](https://lock.cmpxchg8b.com/passmgrs.html#conclusion).
+
+If you can't do this for some reason, then it's OK to use a third-party password manager. This could be necessary if your browser has an accessibility issue, or if you work with multiple browsers.
+
+Some of the third-party password managers people use at GDS include:
+
+- **LastPass** - Approved by Information Assurance and available to install in the GDS Self Service app. However, you can only use the free version on one type of device.
```

Comment:
Perhaps “the free version can only be used on one type of device". As worded, it might be interpreted as "only the free version is permitted to be used" or similar.

Reply:
Because all of the examples below are linked, we should be consistent here too:

```suggestion
- **[LastPass](https://www.lastpass.com/)** - Approved by Information Assurance and available to install in the GDS Self Service app. However, you can only use the free version on one type of device.
```

Reply:
Sure, I've amended with that 👍.